lib/utils/error_handler.dart

```diff
@@ -22,22 +23,52 @@ class ErrorHandler {
 
   static void _flutterErrorBuilder(BuildContext context) {
     FlutterError.onError = (FlutterErrorDetails details) {
-      // debugPrint(details.toString());
+      // debugPrint(details.exception.toString());
 
       /// TODO: Improvise required
       /// The message should not always render what it gets as sometimes it might be too large
-      final errorMessage = details.exception.toString();
-
+      /// Added a maximum size for error message after which it will be truncated
+      String errorMessage = details.exception.toString();
+      int maxSize = 100;
+      if (errorMessage.length > maxSize) {
+        errorMessage =
+            "${details.toStringShort()}:\n${details.exception.toString().substring(0, maxSize)}.....";
```

Comment:
when the error is too long just return maybe Unknown error... incomplete error will cause problem to the user

Like maybe say we encountered an unknown error. If it keeps happening please contact us